queries/elixir/textsubjects-big.scm

```diff
@@ -0,0 +1,6 @@
+(([
+    (call (module) (do_block))
+    (call function: (function_identifier) (call) (do_block))
+    (binary_op left: (identifier) operator: "|>")
+] @_start @_end)
+(#make-range! "range" @_start @_end))
```

Comment:
You can delete this file, these queries were replaced for the slightly better named `textsubjects-container-outer`.